blockchain/block.go

```diff
@@ -15,6 +15,7 @@ type Block struct {
 	PreviousHash string
 	Hash         string
 	Transactions []Transaction
+	Contracts    []SmartContract
```

Comment:
Problem polega na tym, że z perspektywy blockchaina sc i transakcja to w sumie to samo